app/models/project.rb

```diff
@@ -9,13 +9,14 @@ class Project < ActiveRecord::Base
   has_many :orders, through: :loans
   belongs_to :tenant
 
-  validates :description, length: { in: 0..255, allow_nil: false }, presence: true
+  validates :description, length: { in: 100..255, allow_nil: false },
```

Comment:
I think we should allow the description to be longer, but we should limit the length of the title.